src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,56 @@
+.calendar {
+  box-sizing: border-box;
+  max-width: $cell-size * $col-count + $gap * ($col-count - 1) + $padding * 2;
+  margin: auto;
+  padding: $padding;
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+
+  &__day {
+    box-sizing: border-box;
+    width: $cell-size;
+    height: $cell-size;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: $background-color;
+    border: 1px solid $border-color;
+    cursor: pointer;
+    transition: all 0.5s;
```

Comment:
better to use that properties you changed on hover istead of 'all'

Reply:
Tnx for advice)